brink.sh

```diff
@@ -208,10 +208,10 @@ update_path_variables() {
     resolve_python_version
 
     if [ "${OS}" = "win" ] ; then
-        PYTHON_BIN="/lib/python.exe"
+        PYTHON_BIN="/lib/python3.exe"
         PYTHON_LIB="/lib/Lib/"
     else
-        PYTHON_BIN="/bin/python"
```

Comment:
I am not sure this will not break our scripts and build system....we can see.

For SFTPPlus PROD windows, this is not a problem as we rename python.exe to sftpplus-worker.exe

But on Linux, some customers might have scripts calling python.exe directly...

But we can go with python3 and see how it goes.

The idea is that latest brink.sh should be able to read brink.conf and work with both python2 or python3

Reply:
Backtracked on this, as the upstream Windows Python binary is named `python.exe`, not `python3.exe`.